projects/packages/publicize/src/social-image-generator/utilities.php

```diff
@@ -56,10 +58,11 @@ function get_token_body( $text, $image_url, $template ) {
  * @return string|WP_Error  The generated token or a WP_Error object if there's been a problem.
  */
 function fetch_token( $text, $image_url, $template ) {
-	$body            = get_token_body( $text, $image_url, $template );
+	$blog_id         = absint( \Jetpack_Options::get_option( 'id' ) );
+	$body            = get_token_body( $text, $image_url, $template, $blog_id );
 	$rest_controller = new REST_Controller();
 	$response        = Client::wpcom_json_api_request_as_blog(
-		sprintf( 'sites/%d/jetpack-social/generate-image-token', absint( \Jetpack_Options::get_option( 'id' ) ) ),
+		sprintf( 'sites/%d/jetpack-social/generate-image-token', $blog_id ),
```

Comment:
Sorry, I might have led you the wrong way here! I don't think we need to pass in the blog ID from the Jetpack site, because we'll have that on WPCOM, and can include it in the token from there.

Reply:
Oh I see! I can abandon this then